open-reac/src/main/java/com/powsybl/openreac/Reports.java

```diff
@@ -0,0 +1,52 @@
+/**
+ * Copyright (c) 2024, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.powsybl.openreac;
+
+import com.powsybl.commons.reporter.Report;
+import com.powsybl.commons.reporter.Reporter;
+import com.powsybl.commons.reporter.TypedValue;
+
+/**
+ * @author Joris Mancini <joris.mancini_externe at rte-france.com>
+ */
+public final class Reports {
+
+    private Reports() {
+        // Should not be instantiated
+    }
+
+    public static Reporter createVoltageInitReporter(Reporter reporter, String networkId) {
+        return reporter.createSubReporter("voltageInit", "Voltage init on network '${networkId}'", "networkId", networkId);
+    }
+
+    public static void reportConstantQGeneratorsSize(Reporter reporter, int constantQGeneratorsSize) {
+        reporter.report(Report.builder()
+            .withKey("constantQGeneratorsSize")
+            .withDefaultMessage("Reactive power set point is considered fixed for ${size} generators")
+            .withSeverity(TypedValue.INFO_SEVERITY)
+            .withValue("size", constantQGeneratorsSize)
+            .build());
+    }
+
+    public static void reportVariableTwoWindingsTransformersSize(Reporter reporter, int variableTwoWindingsTransformersSize) {
+        reporter.report(Report.builder()
+            .withKey("variableTwoWindingsTransformersSize")
+            .withDefaultMessage("${size} two-windings transformers are considered as variable")
```

Comment:
variable tap positions? Same for shunt variable shunt sections?

Reply:
@annetill like "There are ${size} tap positions considered as variable on two-winding transformers" and "There are ${size} shunt sections considered as variable on shunt compensators" or just "There are ${size} tap positions considered as variable" and "There are ${size} shunt sections considered as variable" ?

Reply:
First proposals.